edge-modules/api-proxy-module/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "api-proxy-module"
+version = "0.1.0"
+authors = ["Azure IoT Edge Devs"]
+license = "MIT"
+edition = "2018"
+
+[dependencies]
+base64 = "0.11"
```

Comment:
I believe we don't need all this dependecies specified here.

Reply:
I think we do, I ran an automated check ("cargo +nightly udeps")
But it flagged another crate that I don't need:
![image](https://user-images.githubusercontent.com/33438817/91107664-a9ccaf00-e62a-11ea-981d-e8555228150f.png)